### PR TITLE
UI - platform/strata dashboard updates

### PR DESCRIPTION
**Pull request by kialj876**
*Issue:*

- bcgov/entity/issues/24327

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: strr-platform-web/app/stores/platform.ts
Comment (deetz99) [inline strr-platform-web/app/stores/platform.ts:1]: nice

**Review by deetz99 — APPROVED** [review]
Looks pretty good to me, nothing glaring. 

Maybe you could walk me through the useStrrBasePermit composable. This is to pull the application/registration details on the detail pages for both platforms and strata, and it auto loads either a registration or application. is that correct?

Also don't forget to update the platform package.json version 👍

File: strr-platform-web/app/pages/platform/dashboard.vue [hunk 1]
```diff
@@ -6,18 +6,19 @@ const accountStore = useConnectAccountStore()
 const { loading, title, subtitles } = storeToRefs(useConnectDetailsHeaderStore())
 const { loadPlatform } = useStrrPlatformStore()
 const {
-  activeApplicationInfo,
-  activePlatform,
+  application,
+  registration,
+  permitDetails,
```
Comment (thorwolpert) [inline strr-platform-web/app/pages/platform/dashboard.vue:11]: nice!

**Review by thorwolpert — APPROVED** [review]
love the move to _permits_

**Comment by kialj876** [comment]
> ## [Quality Gate Failed](https://sonarcloud.io/dashboard?id=bcgov_STRR&pullRequest=299) **Quality Gate failed**
> Failed conditions  [1 Security Hotspot](https://sonarcloud.io/project/security_hotspots?id=bcgov_STRR&pullRequest=299&issueStatuses=OPEN,CONFIRMED&sinceLeakPeriod=true)
> 
> [See analysis details on SonarQube Cloud](https://sonarcloud.io/dashboard?id=bcgov_STRR&pullRequest=299)

validation in the form ensures this is always a valid link even though it is a user entered value

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-strata-hotel-dev--pr-299-x9p2yb1y.web.app

**Review by deetz99 — APPROVED** [review]
Looks good to me 👍

**Pull request merged**